### PR TITLE
Fix PR Preview Playground

### LINKED ISSUE #3581: PR preview playground seems broken
For example: https://deploy-preview-3580--prettier.netlify.com/playground/

image

Probably caused by some changes in #3536, cc @azz.

### PR DESCRIPTION
**Pull request by azz**
Fixes #3581 

Not sure why the name changed from "index" to "prettier".

### CODE REVIEW

**Comment by duailibe** [comment]
Looks like it didn't work :(

File: website/static/worker.js [hunk 1]
```diff
@@ -40,8 +40,11 @@ self.require = function require(path) {
   return self[path];
 };
 
+var prettier;
 importScripts("lib/index.js");
-var prettier = index; // eslint-disable-line
+if (typeof prettier === "undefined") {
+  prettier = index; // eslint-disable-line
```
Comment (ikatyang) [inline website/static/worker.js]: It seems it's called `repo`,  don't know why.

Comment (azz) [reply website/static/worker.js]: Ah, must have been the rollup-plugin-commonjs upgrade.

Comment (azz) [reply website/static/worker.js]: Yep, that worked! :taco:

**Pull request merged**